wagtail/contrib/modeladmin/templates/modeladmin/choose_parent.html

```diff
@@ -20,12 +20,13 @@
 {% block content %}
 
     {% block header %}
-        {% include "modeladmin/includes/header_with_breadcrumb.html" with title=view.get_page_title subtitle=view.get_page_subtitle icon=view.header_icon %}
+        {% include "wagtailadmin/shared/header_with_locale_selector.html" with title=view.get_page_title subtitle=view.get_page_subtitle icon=view.header_icon %}
     {% endblock %}
 
     <div>
 
         <div class="nice-padding">
+            <p class="back"><a href="{{ view.index_url }}" class="icon icon-arrow-left">{% blocktrans trimmed with view.verbose_name as model_name %}Back to {{ model_name }} list{% endblocktrans %}</a></p>
```

Comment:
While we are at it we should use the new svg icon markup (please update the same in the inspect view.

Rough code suggestion below, but not validated.
```suggestion
            <p class="back">
                <a href="{{ view.index_url }}">
                    {% blocktrans trimmed with view.verbose_name as model_name %}Back to {{ model_name }} list{% endblocktrans %}
                    <svg class="icon icon-arrow-right arrow_right_icon" aria-hidden="true">
                        <use href="#icon-arrow-right"></use>
                    </svg>
                </a>
            </p>
```

Reply:
leave is misaligned - let me know when the rest of this PR is good to review and we can get this in.

Please ask at stand-up tonight what is the best practice for using svg icons and to work around issues like this.

cc @thibaudcolas